classes/war/Operation.php

```diff
@@ -49,8 +49,10 @@ class Operation
     const BASE_OPERATION_INTERVAL = 12; // time per interval
     const BASE_OPERATION_COST = 150; // chakra/stam cost per interval, 750 total
     /* 100 / 20 * 12s = 60s */
-    const LOOT_GAIN = 2;
-    const LOOT_OPERATION_SPEED = 100; // eat loot action is only 1 interval
+    const LOOT_GAIN = 1;
+    const LOOT_OPERATION_SPEED = 100; // each loot action is only 1 interval
```

Comment:
Huh, how does 100 correlate to 1 interval?

Reply:
100% progress per interval